From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Paul Gofman <pgofman@codeweavers.com>
Date: Wed, 30 Jul 2025 12:06:03 -0600
Subject: [PATCH] msi: HACK: Report older Windows version for Age of Empires:
 Definitive Edition redists.

CW-Bug-Id: #25520
Valve-commit: 409a76e7c2eee6e79b15bd65f222990890206a2f
Picked: clean
Class: valve
Evidence: docs/evidence/2026-09-28-wine-proton-rebase.md
Offered-upstream: no
---
 dlls/msi/package.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/dlls/msi/package.c b/dlls/msi/package.c
index a07d2c7..5db4c10 100644
--- a/dlls/msi/package.c
+++ b/dlls/msi/package.c
@@ -634,6 +634,7 @@ static void fixup_winver(DWORD *verval)
                     (
                         !strcmp(s, "976730")
                         || !strcmp(s, "231430")
+                        || !strcmp(s, "1017900")
                     );
         if (cached)
             ERR("HACK: setting winver 502.\n");
